runestone/activecode/js/activecode.js

```diff
@@ -357,7 +356,7 @@ ActiveCode.prototype.createGradeSummary = function () {
     var showGradeSummary = function (data, status, whatever) {
         var report = eval(data)[0];
         // check for report['message']
-        if (report) {
```

Comment:
This section undoes changes I made a few weeks ago and has nothing to do with styles.